### DISCUSSION TITLE
Error compiling benchmark

### DISCUSSION THREAD

**Discussion by themagicm** [moononournation/Arduino_GFX#259] [Q&A]
ESP32-8048S070, Arduino IDE, LVGL 8.3.4, Benchmark enabled in lv_conf.sh

 multiple definition of `lv_demo_benchmark_set_max_speed'; /tmp/arduino-sketch-604128586A1AEBABB56B936B9A9C4EC7/sketch/objs.a(lv_demo_benchmark.c.o):/tmp/.arduinoIDE-unsaved202318-6748-98xp8t.ucccq/LvglBenchmark/lv_demo_benchmark.c:775: first defined here
collect2: error: ld returned 1 exit status

**Comment by moononournation** [comment]
then try not enable benchmark

**Reply by themagicm** [reply]
not enabling benchmark in lv_conf.h:
/tmp/.arduinoIDE-unsaved202318-36561-wc5qxb.igyyf/LvglBenchmark/LvglBenchmark.ino:143: undefined reference to `lv_demo_benchmark'

enabling benchmark in lv_conf.h:
/libraries/lvgl/src/demos/benchmark/lv_demo_benchmark.c:748: multiple definition of `lv_demo_benchmark_set_max_speed'; /tmp/arduino-sketch-66EE15C87FD8D6A8587687AA99C9A980/sketch/objs.a(lv_demo_benchmark.c.o):/tmp/.arduinoIDE-unsaved202318-36561-wc5qxb.igyyf/LvglBenchmark/lv_demo_benchmark.c:775: first defined here
collect2: error: ld returned 1 exit status

**Reply by moononournation** [reply]
try start from the lv config template again
